.pre-commit-config.yaml

```diff
@@ -14,3 +14,7 @@ repos:
     rev: 20.8b1
     hooks:
     -   id: black
```

Comment:
Do we still need black / autopep? Not a problem if we do, just checking!

Reply:
I think we still do so future PRs have the same formatting that black/autopep suggest! Black has a very specific code style so enforcing that going forward makes the code more readable

Reply:
That makes sense.